### PR TITLE
added a README file for BSP tests

### PR DESCRIPTION
**Pull request by akarambalkar**
I have created a new README.md file explaining in detail the required pre-condtion creation and test execution steps for BSP test scenarios. Please review and let me know if any changes required.

### CODE REVIEW

File: tests/BSP/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	BSP (Tests for Business, BusinessPlus, Standard and Professional editions of zimbra-X)
+#Overview
+
+BSP Test suite defined to simulate customer environment for load testing using different user types. We have designed this test to hit the required load for BSP customer type. In this test suite we will load test against different types of BSP editions of Zimbra-X as in Professional, Standard, BusinessPlus and Business editions.
+
+Desired load for SOAP, IMAP and POP will depend on the requirement
+
+#Prerequisites - Zimbra-X
+
+When you are deploying Zimbra-X, there are some additional global config options that should be set so that the BSP test will run properly. Here is a quick walk-through. It assumes that youhave created a deployment branch in zm-kubernetes that is based on the latest develop branch. The walk-through does not cover all possible options and settings that you may need. It documents just the configuration needed for the BSP tests.
```
Comment (thefoolwhocodes) [inline tests/BSP/README.md]: Why a deployment branch in zm-kubernetes is must for this?

File: tests/BSP/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Before deploying (select the value for TEMPLATE_ENV that you desire)
```
Comment (thefoolwhocodes) [inline tests/BSP/README.md]: Give what are options for TEMPLATE_ENV and what is preferrable.

File: tests/BSP/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+export TEMPLATE_ENV=templates/base.env
+
+make TEMPLATE_ENV=${TEMPLATE_ENV} clean init-keys init-configs init-passwords
```
Comment (thefoolwhocodes) [inline tests/BSP/README.md]: Can i just do `make initialize` instead of above two lines.

File: tests/BSP/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+Then execute the below command to create a COS for  Business Email Edition.
+
+zmprov cc Business zimbraFeatureMAPIConnectorEnabled FALSE zimbraFeatureMobileSyncEnabled FALSE zimbraArchiveEnabled FALSE zimbraFeatureConversationsEnabled FALSE  zimbraFeatureTaggingEnabled FALSE zimbraAttachmentsIndexingEnabled FALSE  zimbraFeatureViewInHtmlEnabled FALSE zimbraFeatureGroupCalendarEnabled FALSE zimbraFreebusyExchangeURL FALSE zimbraFeatureSharingEnabled FALSE zimbraFeatureTasksEnabled FALSE zimbraFeatureBriefcasesEnabled FALSE zimbraFeatureSMIMEEnabled FALSE zimbraFeatureVoiceEnabled FALSE zimbraFeatureManageZimlets FALSE zimbraFeatureCalendarEnabled FALSE zimbraFeatureGalEnabled TRUE
+
+In response you will receive a alpha numeric cos id something like this < 0a9d288d-d1e6-4f6b-a1a6-05b1e98daf57 > save this cos ID 
+
+Then using above recieved cos ID for Business edition we need to create users in the database to provide access exclusively to zimbra-X Business edition.
+
+Use the below command to create users
```
Comment (thefoolwhocodes) [inline tests/BSP/README.md]: Explain how many users below command creates, and what needs to be changed if we need to create more users.

File: tests/BSP/README.md (hunk 1 above)
Comment (akarambalkar) [reply tests/BSP/README.md]: Updated document

File: tests/BSP/README.md (hunk 3 above)
Comment (akarambalkar) [reply tests/BSP/README.md]: Updated Document

File: tests/BSP/README.md (hunk 4 above)
Comment (akarambalkar) [reply tests/BSP/README.md]: Updated comment

File: tests/BSP/README.md (hunk 2 above)
Comment (akarambalkar) [reply tests/BSP/README.md]: Updated Document

**Pull request merged**